js/interactions.md

```diff
@@ -12,13 +12,50 @@ Ensure you have [installed and configured the Amplify CLI and library]({%if jeky
 
 AWS Amplify implements [Amazon Lex](https://aws.amazon.com/lex) as the default chatbots service. Amazon Lex supports creating conversational bots with the by the same deep learning technologies that power Amazon Alexa.
 
-## Create your Chatbot
+#### Automated Setup
+
+Run the following command in your project's root folder:
+
+```bash
+$ amplify add interactions
+```
+
+The CLI will lead you through the steps to specify the chatbot to be created. 
+
+You can choose to start from a sample chatbot or start from scratch.  If you choose to start from scratch, the CLI will prompt you with a series of questions to set the intents and slots for the chatbot. 
+
+You are allowed to run the `amplify add interactions` command multiple times to add multiple chatbots into your project.
+
+{The Interactions category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.}
+{: .callout .callout--info}
+
+The `add` command automatically creates a backend configuration locally. To update your backend in the cloud, run:
+
+```bash
+$ amplify push
+```
+
+Upon successful execution of the push command, a configuration file called `aws-exports.js` will be copied to your configured source directory, for example `./src`. 
+
+##### Configure Your App
+
+Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point. For example `App.js` in React or `main.ts` in Angular.
+
+```js
+import Amplify, { Interactions } from 'aws-amplify';
+import aws_exports from './aws-exports';
+
+Amplify.configure(aws_exports);
+```
+
+### Manual Setup
+
+#### Create your Chatbot
 
 You can create Amazon Lex chatbox in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex Developer Guide](https://docs.aws.amazon.com/lex/latest/dg/getting-started.html).
  
 ![Interactions]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/media/images/interactions_lex_console_edit_bot.jpg){: class="screencap" style="max-height:600px;"}
```

Comment:
This is another bug - ![Interactions]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/media/images/interactions_lex_console_edit_bot.jpg){: class="screencap" style="max-height:600px;"